packages/sdk-router/src/constants/chainIds.ts

```
@@ -33,13 +33,19 @@ const UNSUPPORTED_BRIDGE_CHAIN_IDS: number[] = [
 
+/**
+ * List of paused chain IDs
+ */
+export const PAUSED_CHAIN_IDS: SupportedChainId[] = [SupportedChainId.BOBA]
+
 /**
  * List of supported chain ids, where SynapseBridge is deployed.
  */
 export const SUPPORTED_CHAIN_IDS: number[] = Object.values(SupportedChainId)
   .map((chainId) => Number(chainId))
   .filter((chainId) => !isNaN(chainId))
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)